test/AclTest.php

```diff
@@ -1466,18 +1466,50 @@ public function testAllowNullPermissionAfterResourcesExistShouldAllowAllPermissi
     /**
      * @see https://github.com/laminas/laminas-permissions-acl/issues/2
      */
-    public function testWhenAssertionReturnsFalseTheInversionOfItsTypeShouldBeUsed()
+    public function testCanDenyAnInheritedRuleThatAllows()
```

Comment:
```suggestion
    public function testCanDenyAnInheritedAllowRule()
```

As a native English speaker, I read this and thought "allows what?"; the suggested change makes it more clear that you are denying a previously set rule.